backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -1,170 +1,225 @@
-                {% else %}
-                <p class="text-sm">
-                    {{card.due_time|naturalday}}, {{card.due_time|time}} 
-                </p>
+    <!-- Flavours section -->
+
+    {% if card.flavour_names %}
+    <div class="grid grid-cols-1 gap-y-0 w-full h-[auto] pb-2">
+        {% for flavour_name in card.flavour_names %}
+            {%if flavour_name != "none"%}
+                {%if flavour_name == "javascript"%}
+                    <span title="{{flavour_name}}">
+                        <i class="fa-regular fa-brands fa-js fa-lg"></i>
+                    </span> 
+                {%endif%}
+                {%if flavour_name == "python"%}
+                    <span title="{{flavour_name}}">
+                        <i class="fa-regular fa-brands fa-python fa-lg"></i>
+                    </span> 
+                {%endif%}
+                {%if flavour_name == "java"%}
+                    <span title="{{flavour_name}}">
+                        <i class="fa-regular fa-brands fa-java fa-lg"></i>
+                    </span> 
+                {%endif%}
             {% endif %}
-        </div>
-    {% endif %}
+        {% endfor %}
+    </div>
     
-    {% if card.request_user_is_assignee == False %}
-        <div class="w-full" >
-            <p class="text-xs font-bold text-gray-600">Assignee</p>
-            <p class="truncate" title="{{card.assignee_names.0}}">{{card.assignee_names.0}}</p> 
-        </div>
     {% endif %}
 
-    {% if card.reviewer_names %}
-        <div class="w-full">
-            <p class="text-xs font-bold text-gray-600">Reviewers</p>
-            {% for reviewer_name in card.reviewer_names %}
-                <p class="truncate" title="{{reviewer_name}}">{{reviewer_name}}</p> 
-            {% endfor %}
-        </div>
-    {% endif %}
+    <!-- Horizontal line -->
+ 
+    <hr class="border-gray-300 inset-x-0 absolute" />
 
-    {% if card.repo_url %}
-        <div class="flex gap-2 items-center">
-            <i class="fa-regular fa-brands fa-github fa-lg"></i>
-            <a class="{{styles.link}}" target="_blank" href="{{card.recruit_project.repository.get_github_repo_link}}">Repository</a>  
-        </div>
-    {% endif %}
+    <!-- Assignee and reviewers section -->
 
-    {% if card.request_user_can_start %}
-        <button type="button" class="{{ styles.button_primary_small }}"
+    <div class="grid grid-cols-2 grid-rows-1 gap-x-10 gap-y-3 w-full h-[auto] pt-2">
+
+      <div class="Assignee-avatar col-span-1 row-span-1 group inline-block">
+        <div class="mb-2 text-xs text-indigo-950 font-['Poppins']">Assignee</div>
+        {% if card.assignee_names %}
+          {% for assignee in card.assignee_names %}
+            <img
+              class="w-[30px] h-[30px] rounded-full"
+              src="https://images.unsplash.com/photo-1594751543129-6701ad444259?q=80&w=1000&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Mnx8ZGFyayUyMHByb2ZpbGV8ZW58MHx8MHx8fDA%3D"
```

Comment:
for now can you please make this an anonomous image/icon. We are unlikely to implement profile pics before we go live.